hudi-client/hudi-flink-client/pom.xml

```diff
@@ -260,6 +260,17 @@
                 <groupId>org.apache.rat</groupId>
                 <artifactId>apache-rat-plugin</artifactId>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <forkMode>always</forkMode> <!-- Required for environment variables to work reliably -->
```

Comment:
is it possible to set this for specific needing test case class? instead of changing the whole module's test setup? any side-effect of using fork mode for other tests in the module?